### PR TITLE
Add a test for assembly level attributes in NetModules in absence of AssemblyAttributesGoHere types.

### PR DESCRIPTION
**Pull request by AlekseyTs**
Related to #10550.

@dotnet/roslyn-compiler Please review.

### CODE REVIEW

File: src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_Assembly.cs [hunk 1]
```diff
@@ -1056,6 +1058,118 @@ class Program
...
+            Assert.False(token.IsNil);   //could the type ref be located? If not then the attribute's not there.
+
+            var consoleappCompilation = CreateCompilation(
+                consoleappSource,
+                references: new[] { MinCorlibRef, netModuleWithAssemblyAttributes.GetReference() },
+                options: TestOptions.ReleaseExe);
+
+            Assert.Null(consoleappCompilation.GetTypeByMetadataName("System.Runtime.CompilerServices.AssemblyAttributesGoHere"));
+            Assert.Null(consoleappCompilation.GetTypeByMetadataName("System.Runtime.CompilerServices.AssemblyAttributesGoHereM"));
+            var diagnostics = consoleappCompilation.GetDiagnostics();
```
Comment (cston) [inline src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_Assembly.cs]: Can we assert anything meaningful about `diagnostics`?

**Comment by cston** [comment]
LGTM

File: src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_Assembly.cs (hunk 1 above)
Comment (AlekseyTs) [inline src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_Assembly.cs]: Sure, I'll add a check.

**Comment by AlekseyTs** [comment]
Merging with a single sign-off since it is a test only change.

**Pull request merged**